pkg/services/system_intakes_test.go

```diff
@@ -46,6 +46,51 @@ func (s ServicesTestSuite) TestSystemIntakesByEuaIDFetcher() {
 	})
 }
 
+func (s ServicesTestSuite) TestSystemIntakeCreator() {
```

Comment:
Can we try to name tests after function names? It's pretty common go pattern, and mostly used in this code base. `TestNewCreateSystemIntake` in this case I believe.